assets/stylesheets/mobile.css

```diff
@@ -0,0 +1,460 @@
+  font-size: 17px;
+  font-weight: bold;
+  color: #d3d3d3;
+  font-family: 'Lato', sans-serif;
+}
+
+footer hr,
+.main-program hr,
+.tutors hr,
+.testimonies hr,
+.logo-descript hr {
+  width: 40px;
+  border: 1px solid #ec5242;
+}
+
+.program-list {
+  list-style-type: none;
+  padding: 0;
+}
+
+.flex1,
+.flex2,
+.flex3 {
+  margin: 3px;
+  font-size: 14px;
+  font-weight: 600;
+  font-family: 'Lato', sans-serif;
+}
+
+.flex1 {
+  width: 48px;
+  height: 48px;
+}
+
+.flex2 {
+  min-width: 25%;
+  overflow: hidden;
+  color: #ec5242;
+}
+
+.flex3 {
+  min-width: 50%;
+  overflow: hidden;
+  color: #272a31;
+}
+
+.join-gst {
+  display: block;
+  margin: 0 auto;
+  border: none;
+  background: #ec5242;
+  padding: 20px 30px;
+  color: white;
+  font-family: 'Lato', sans-serif;
+  font-size: 20px;
+  margin-top: 30px;
+}
+
+/******************* Tutors *****************/
+
+.tutors {
+  padding-left: 6%;
+  padding-right: 6%;
+}
+
+.tutors-title {
+  text-align: center;
+  font-weight: bold;
+  color: red;
+  font-family: 'Lato', sans-serif;
+}
+
+.grid-profiles {
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.tutor2 ~ .grid-profiles {
+  display: none;
+}
+
+.show-less .tutor2 ~ .grid-profiles {
+  display: block;
+}
+
+.grid-pro-1 {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding-left: 12px;
+  padding-top: 12px;
+  object-fit: cover;
+  object-position: bottom;
+}
+
+.image-container {
+  background-image: url('../images/chess.jpg');
+  background-size: 40% 40%;
+  background-repeat: no-repeat;
+  background-position-x: 0;
+  background-position-y: 0;
+  padding: 0;
+}
+
+.grid-pro-2 {
+  margin: 0;
+  font-weight: bold;
+  word-break: break-all;
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+}
+
+.grid-pro-3 {
+  margin: 0;
+  font-size: 12px;
+  font-style: italic;
+  color: #ec5242;
+  font-weight: bold;
+  word-break: break-all;
+  font-family: 'Lato', sans-serif;
+  line-height: 20px;
+}
+
+.grid-pro-4 {
+  word-break: break-all;
+  margin: 0;
+  font-family: 'Lato', sans-serif;
+  font-size: 13px;
+  line-height: 20px;
+  font-weight: bold;
+}
+
+.profile-info::after {
+  content: "";
+  clear: both;
+  display: block;
+}
+
+.profile-info {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  float: right;
+  clear: both;
+  padding-top: 5px;
+}
+
+.grid-profiles p {
+  margin-left: 10px;
+  padding: 3px;
+}
+
+.more-tutors,
+.show-less {
+  border: 2px solid gray;
+  border-radius: 3px;
+  text-align: center;
+  width: 100%;
+  background-color: white;
+  padding: 8px;
+}
+
+.more-tutors i,
+.show-less i {
+  color: orange;
+}
+
+.show-more-courses {
+  display: none;
+}
+
+.mobile-footer {
+  display: none;
+}
+
+/************************ About Page ************************/
+
+.body {
+  background-color: #e6e4e4;
+}
+
+.description {
+  background-color: #e6e4e4;
+}
+
+.description h1 {
+  color: orangered;
```

Comment:
- [ ] In the Behance design, the designer set a specific color palette. In this case, for the orange, it has the following value: `#EC5242`. You should consider changing it to match the given design.
- [ ] Beware that this also other times in your code!